fetch-publication/src/main/java/no/unit/nva/publication/fetch/FetchPublicationHandler.java

```diff
@@ -15,14 +17,13 @@
 import nva.commons.utils.Environment;
 import nva.commons.utils.JacocoGenerated;
 import org.apache.http.HttpStatus;
-
-import static nva.commons.utils.JsonUtils.objectMapper;
-
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class FetchPublicationHandler extends ApiGatewayHandler<Void, JsonNode> {
 
     public static final String PUBLICATION_CONTEXT_JSON = "publicationContext.json";
-
+    private static final Logger LOGGER = LoggerFactory.getLogger(FetchPublicationHandler.class);
```

Comment:
I think the all caps names should be reserved for constants.

Reply:
Codestyle complains since it is private static final